pkg/p2p/specwrapper.go

```diff
@@ -0,0 +1,27 @@
+package p2p
+
+import (
+	"context"
+	"errors"
+	"time"
+)
+
+var ErrUnexpected = errors.New("unexpected request while in light mode")
+
+// WithDisconnectStreams will mutate the given spec and replace the handler with a always erroring one
```

Comment:
godoc comments should end with a period :)

Reply:
fixed, should we enable the linter for this one?